ceph/ceph.py

```diff
@@ -187,6 +187,9 @@ def setup_ceph_firewall(self):
                 node.configure_firewall()
                 node.open_firewall_port(port=ports, protocol="tcp")
 
+            if node.role == "rbd-mirror":
```

Comment:
@psathyan rbd-mirror daemon don't need any ports to be opened so I've added a separate condition
otherwise we could add configure_firewall (install and enable firewall irrespective of role) but it would be redundant on nodes like client or installer so I did not make configure_firewall as generic call. 

Can you please let me know your thoughts on making it something like 
`
-            if ports:
+ 
+            if node.role not in ['_admin','installer','client]:
                 node.configure_firewall()
+
+            if ports:
                 node.open_firewall_port(port=ports, protocol="tcp")
`